profiler/matchers.rst

```diff
@@ -14,18 +14,26 @@ that feature was removed in Symfony 4.0. However, you can use the ``enable()``
 and ``disable()`` methods of the :class:`Symfony\\Component\\HttpKernel\\Profiler\\Profiler`
 class in your controllers to manage the profiler programmatically::
 
-    use Symfony\Component\HttpKernel\Profiler\Profiler;
+    use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
     // ...
 
-    class DefaultController
+    class DefaultController extends AbstractController
     {
         // ...
 
-        public function someMethod(Profiler $profiler)
+        public function someMethod()
         {
             // for this particular controller action, the profiler is disabled
-            $profiler->disable();
+            $this->container->get('profiler')->disable();
```

Comment:
This almost asks for a specific feature to turn it off in an elegant way. I do not agree with the service locater though, I would rather see the service id commented above it with a note that it can't be autowired.